pkgs/top-level/all-packages.nix

```diff
@@ -16902,6 +16902,9 @@ with pkgs;
 
   opensmalltalk-vm = callPackage ../development/compilers/opensmalltalk-vm { };
 
+  opensycl = callPackage ../development/compilers/opensycl { };
```

Comment:
`aligned_alloc` was added in 10.15, so you need to use the 11.0 SDK.

```suggestion
  opensycl = darwin.apple_sdk_11_0.callPackage ../development/compilers/opensycl { };
```

Reply:
Thank you very much :D
it appears to work for me on linux too, which I find odd, since it specifically says something about darwin, but hey, as long as it works.

Reply:
It delegates to the default stdenv on Linux.